README.rst

```diff
@@ -1,207 +1,65 @@
 SymbiFlow examples
 ==================
 
-This repository provides example FPGA designs that can be built using the SymbiFlow open source toolchain.
-The examples target the Xilinx Artix-7 and the QuickLogic EOS S3 devices.
+.. image:: https://travis-ci.com/SymbiFlow/symbiflow-examples.svg?branch=master
```

Comment:
Travis is no longer used right? This is all on GitHub actions now?

Reply:
fixed